werkzeug/wrappers.py

```diff
@@ -1233,9 +1233,9 @@ def get_wsgi_headers(self, environ):
         # Also update content_length accordingly so that the automatic
         # content length detection does not trigger in the following
         # code.
-        if 100 <= status < 200 or status == 204:
+        if 100 <= status < 200:
             headers['Content-Length'] = content_length = u'0'
-        elif status == 304:
+        elif status == 304 or status == 204:
```

Comment:
@ThiefMaster True. We create a frozen set once, then just refer to it in the `if` condition; I simplified the example above for discussion's sake.

Nevertheless, I checked the bytecode, and you're right; for a small number of values, it's probably more efficient to use `x in (1, 2)`. Thanks--I'm happy to have learned this!

I'm happy to make the change, but it sounds like this PR is going to be closed in favor of #1026, anyway, so I'll just leave it as is unless I hear otherwise. Thanks again.